Lib/test/test__xxsubinterpreters.py

```diff
@@ -759,7 +759,11 @@ def test_still_running(self):
         main, = interpreters.list_all()
         interp = interpreters.create()
         with _running(interp):
-            with self.assertRaises(RuntimeError):
+            self.assertTrue(interpreters.is_running(interp),
+                msg=f"Interp {interp} should be running before destruction.")
```

Comment:
Typically we want to line up additional "arg lines" with the first "arg line".  So in this case the `msg=` line should line up the first arg, right after the `assertTrue(`, like this:

```suggestion
                            msg=f"Interp {interp} should be running before destruction.")
```